lib/game.rb

```diff
@@ -0,0 +1,68 @@
+require 'pry'
+class Game
+
+	attr_accessor :board, :player_1, :player_2
+	extend Players
+
+	WIN_COMBINATIONS = [
+    [0,1,2],[3,4,5],[6,7,8],
+    [0,3,6],[1,4,7],[2,5,8],
+    [0,4,8],[2,4,6]]
+
+    def initialize(player_1 = Players::Human.new('X'), player_2 = Players::Human.new('O'), board=Board.new)
+    	@player_1 = player_1
+    	@player_2 = player_2
+    	@board = board
+    end
+
+    def current_player
+    	count = self.board.turn_count
+    	return self.player_1 if count.even?
+    	return self.player_2 if count.odd?
+    end
+
+    def over?
+    	self.board.full? || self.won?
+    end
+
+    def won?
+    	WIN_COMBINATIONS.detect do |array|
+      	self.board.cells[array[0]] == self.board.cells[array[1]] && self.board.cells[array[1]] == self.board.cells[array[2]] && (self.board.cells[array[0]]=='X' ||self.board.cells[array[0]]=='O')
+    	end
+    end
+
+    def draw?
+    	self.over? && !self.won?
+    end
+
+    def winner
+    	if self.won?
```

Comment:
Instead of calling `Game#won?` twice, consider using an if statement assignment. For example:

```ruby
if won = won?
  board.cells[won.first]
end
```

If `won?` returns true, then and only then will the local variable `won` be assigned, otherwise, the method will just abort with a false return value. But if `won` is assigned, we can use that variable instead of calling `won?` again within the if statement.